edi_voxel_stock_picking_oca/models/stock_picking.py

```diff
@@ -75,9 +75,14 @@ def able_to_voxel(record):
 
         pickings = self.filtered(able_to_voxel)
         if pickings:
-            pickings.enqueue_voxel_report(
-                "edi_voxel_stock_picking_oca.report_voxel_picking"
-            )
+            for picking in pickings:
+                report_id = picking.partner_id.voxel_report_id
+                report = (
+                    report_id.get_external_id().get(report_id.id)
+                    if report_id
+                    else "edi_voxel_stock_picking_oca.report_voxel_picking"
+                )
```

Comment:
- Name properly the variables. It should be `report`, as it contains a recordset, not the integer ID.
- Doing an `or`, you also cover the case where there's no XML-ID associated with the selected report.

```suggestion
                report = picking.partner_id.voxel_report_id
                report = ( 
                    report.get_external_id().get(report_id.id)
                    or
                    "edi_voxel_stock_picking_oca.report_voxel_picking"
                )
```

Anyway, this has a problem: it's not accepting reports manually created without XML-ID. You can annotate that problem to be taken into account.